README.md

```diff
@@ -1,2 +1,37 @@
+
+Microverse
+
 # Hello-World
 setup of my first microvesre project
+
+
+Built With
+Major languages:css and html
+Technologies used:github,git,linters
+
+
+Getting Started
+
+
+Prerequisites
+
+Install:you need to clone the repository  to your local computer.
+Usage:this is just the first microverse project to check all fundamentals to pass code review.
+
+👤 Author
+
+GitHub: @ mugishasam123
+Twitter: @mugishasamuel
+LinkedIn: mugishasamuel
```

Comment:
- [ ] As recommended I would suggest updating this to have links to the correct sites. Please check this [Markdown Cheat Sheet](https://www.markdownguide.org/cheat-sheet/) to see how you can give your README.md some cool format :v: